final_server.py

```
@@ -7,6 +7,12 @@
 import json_parser
 
 
+class MoreFilesThanExcpectedException(Exception):
+    pass
```

you should return the Exception init in a function, instand of pass

---

V